manifests/service.pp

```diff
@@ -3,22 +3,19 @@
   $passenger = $::foreman::passenger,
   $app_root  = $::foreman::app_root,
 ) {
-  case $passenger {
-    true: {
-      exec {'restart_foreman':
-        command     => "/bin/touch ${app_root}/tmp/restart.txt",
-        refreshonly => true,
-        cwd         => $app_root,
-        path        => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
-      }
-
-      $service_ensure = 'stopped'
-      $service_enabled = false
-    }
-    default, false: {
-      $service_ensure  = 'running'
-      $service_enabled = true
+  if $passenger {
+    exec {'restart_foreman':
+      command     => "/bin/touch ${app_root}/tmp/restart.txt",
```

Comment:
That was in puppet-puppet for the puppet master restart, but can be applied here as well.